[2025-08-24T09:29:56.141+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: poc_pipeline.process_upload manual__2025-08-24T09:29:52.437175+00:00 [queued]>
[2025-08-24T09:29:56.148+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: poc_pipeline.process_upload manual__2025-08-24T09:29:52.437175+00:00 [queued]>
[2025-08-24T09:29:56.149+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2025-08-24T09:29:56.164+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): process_upload> on 2025-08-24 09:29:52.437175+00:00
[2025-08-24T09:29:56.178+0000] {standard_task_runner.py:60} INFO - Started process 3184 to run task
[2025-08-24T09:29:56.196+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'poc_pipeline', 'process_upload', 'manual__2025-08-24T09:29:52.437175+00:00', '--job-id', '20', '--raw', '--subdir', 'DAGS_FOLDER/poc_pipeline.py', '--cfg-path', '/tmp/tmpaecsfbem']
[2025-08-24T09:29:56.216+0000] {standard_task_runner.py:88} INFO - Job 20: Subtask process_upload
[2025-08-24T09:29:56.307+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:194 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-08-24T09:29:56.480+0000] {task_command.py:423} INFO - Running <TaskInstance: poc_pipeline.process_upload manual__2025-08-24T09:29:52.437175+00:00 [running]> on host 046d27fa1875
[2025-08-24T09:29:56.689+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='poc_pipeline' AIRFLOW_CTX_TASK_ID='process_upload' AIRFLOW_CTX_EXECUTION_DATE='2025-08-24T09:29:52.437175+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-08-24T09:29:52.437175+00:00'
[2025-08-24T09:29:56.694+0000] {logging_mixin.py:188} INFO - [Dummy Mode] No real upload found. Using generated upload_id=dummy-b3248e3f-b142-4ecd-9001-4d659c19eb40
[2025-08-24T09:29:56.695+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-08-24T09:29:56.709+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=poc_pipeline, task_id=process_upload, execution_date=20250824T092952, start_date=20250824T092956, end_date=20250824T092956
[2025-08-24T09:29:56.756+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-08-24T09:29:56.789+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
